vector/vector.yaml

```diff
@@ -27,10 +27,11 @@ transforms:
 sinks:
   # Send logs to Papertrail
   papertrail:
-    type: papertrail
-    inputs:
-      - papertrail_services_remapped
-    endpoint: '${PAPERTRAIL_HOST_AND_PORT}'
     encoding:
       codec: text
+    endpoint: '${PAPERTRAIL_HOST_AND_PORT}'
```

Comment:
(I also alphabetized the other configuration entries. The only substantive change is adding the `if` below.)